addons/misc/functions/fnc_FAK_displayContent.sqf

```diff
@@ -0,0 +1,105 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Miss Heda
+ * Displays FAK items to player
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ * 1: FAK Type <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, 0] call kat_misc_fnc_FAK_displayContent;
+ *
+ * Public: No
+ */
+
+params ["_unit", "_FAKType"];
+
+if (_unit != ACE_player) exitWith {};
+
+private _FAKContent = "";
+private _outputString = "";
+private _mainOutputString = "";
+private _ultraMainOutputString = "";
+private _slot = 0;
+private _firstnline = "";
+private _nline = "<br/>";
+private _itemColor = "";
+private _slotColor = "";
+private _slotNameArray = missionNameSpace getVariable [QGVAR(FAKSlotNames), []];
+private _slotName = "";
+
+private _fnc_arrayToHexCode = {
+    params ["_array"];
+
+    private _return = "";
+    private _hashtag = "";
+    private _hexCode = "";
+    private _count = 0;
+
+    {
+        _count = _count + 1;
+        _x = _x * 255;
+        _hexCode  = [_x] call ace_common_fnc_toHex;
+        if (_count == 3) then { _hashtag = "#"; };
+        _return = _hashtag + _return + _hexCode; 
+    } forEach _array; //Get correct Hex color code
+    
+    _return
+};
+
+switch (_FAKType) do {
+    case 0: { 
+        _FAKContent = missionNameSpace getVariable [QGVAR(IFAKContents), []];
+        _slotColor = [(missionNameSpace getVariable [QGVAR(IFAK_Slot_Color), []])] call _fnc_arrayToHexCode;
+        _itemColor = [(missionNameSpace getVariable [QGVAR(IFAK_Item_Color), []])] call _fnc_arrayToHexCode;
+    };
+    case 1: { 
+        _FAKContent = missionNameSpace getVariable [QGVAR(AFAKContents), []];
```

Comment:
variable naming. First letter lowercase (wont mark everyone of them)

Reply:
We are using _FAK... in other files like that as well. See repack & pack fnc. I just copied what Blue established when rewriting the FNCs. I would rather keep this consistent for FAK variables instead of having it different in other files. And writing it upercase to let the viewer know that a FAK is meant does not affect the functionality of the var what so ever.